lib/action_controller/serialization.rb

```diff
@@ -18,6 +18,10 @@ def serialization_scope(scope)
       self._serialization_scope = :current_user
     end
 
+    def namespace_for_serializer
+      @namespace_for_serializer ||= self.class.parent unless self.class.parent == Object
+    end
```

Comment:
I'm sure people would appreciate it. And I don't think it would be a problem to support, as tons of people want namespaced-scoped serializer lookup -- but maybe that'd be better for the lookup customization PR?